libs/blog/roadmap/feature-roadmap/src/lib/feature-roadmap.component.ts

```
@@ -0,0 +1,214 @@
+      title: 'Sass',
+    },
+    {
+      id: '8',
+      previousNodeId: '2',
+      title: 'Modules',
+    },
+  ]);
+
+  protected readonly roadmapLayers = computed<RoadmapLayer[]>(() => {
+    const nodeDtoMap = this.nodesDto().reduce(
+      (acc, node) => ({ ...acc, [node.id]: node }),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)